### PR TITLE
Kotlin Portlet - Removed Maven

### PR DESCRIPTION
**Pull request by vishal-reddy**
n Kotlin Portlet removed due to compiler JavaC compiler differences + Rebased

### CODE REVIEW

**Comment by gamerson** [comment]
jenkins retest

File: gradle/settings.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ include(
 	'apps:greedy-policy-option-portlet:service-reference',
 	'apps:jsp-portlet',
 	'apps:jsp-war-portlet',
+    'apps:kotlin-portlet',
```
Comment (gamerson) [inline gradle/settings.gradle]: Check whitespace here.

File: maven/apps/pom.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
 		<module>greedy-policy-option-portlet</module>
 		<module>jsp-portlet</module>
 		<module>jsp-war-portlet</module>
+		<module>kotlin-portlet</module>
```
Comment (gamerson) [inline maven/apps/pom.xml]: need to remove this here.

**Review by gamerson — CHANGES_REQUESTED** [review]

**Comment by vishal-reddy** [comment]
jenkins retest

**Comment by vishal-reddy** [comment]
jenkins retest

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/390. [See changes here.](https://github.com/gamerson/liferay-blade-samples/compare/bc52b7c~3...bc52b7c) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**